web/src/Components/CommentsZone/CommentsZone.css

```diff
@@ -0,0 +1,56 @@
+.full-comments-zone {
+	border-top: 1px dashed #CCC;
+}
+
+.comments-zone {
+	height: 310px;
```

Comment:
A fixed height will make the container scroll even if it is not at full height (see screenshot). Sure you want this?
![screenshot from 2018-05-11 21-08-54](https://user-images.githubusercontent.com/5544363/39942358-8c217a48-555f-11e8-967e-2f2d42260cbe.png)